web/src/routes/events/components/EventListItem.js

```diff
@@ -7,18 +7,15 @@ import EventPlaceholder1 from '../assets/EventPlaceholder1.jpg'
 import EventPlaceholder2 from '../assets/EventPlaceholder2.jpg'
 import EventPlaceholder3 from '../assets/EventPlaceholder3.jpg'
 import DateFormatter from 'api-client/src/i18n/DateFormatter'
+import textTruncator from '../../../modules/common/utils/textTruncator'
 
-const EXCERPT_LENGTH = 70
+export const NUM_OF_WORDS_ALLOWED = 8
```

Comment:
I think we could allow a few more words, what do you think?

Reply:
I think a few more words are okay, if the content does not contain to many linebreaks. We could also filter linebreaks in the text truncator. Then we can easily set the wordcount to 15